tests/foreman/cli/test_host.py

```diff
@@ -1880,21 +1875,36 @@ def test_positive_erratum_applicability(katello_host_tools_client):
 
 @pytest.mark.katello_host_tools
 @pytest.mark.tier3
-def test_negative_install_package(katello_host_tools_client):
-    """Attempt to install a package to a host remotely
+def test_positive_install_package_via_rex(
```

Comment:
is this not a test we already have?

Reply:
changed the description, 
and we have [one test](https://github.com/SatelliteQE/robottelo/blob/6a596f61b60e546efa55f488177ea41722e7c9b9/tests/foreman/cli/test_remoteexecution.py#L351) that installs multiple packages the same way I did, but no test that is using `hammer host package list` 

To differentiate the test even more I update the job invocation command, instead of `job-template` I used `remote-execution-feature`